server/elasticsearch/src/test/java/org/hawkular/apm/server/elasticsearch/AnalyticsServiceElasticsearchTest.java

```diff
@@ -2828,7 +2829,7 @@ public void testGetCommunicationSummaryStatisticForHost() {
         criteria.setStartTime(1000).setEndTime(10000);
         criteria.setHostName("hostA");
 
-        Wait.until(() -> analytics.getCommunicationSummaryStatistics(null, criteria, false).size() == 2);
+//        Wait.until(() -> analytics.getCommunicationSummaryStatistics(null, criteria, false).size() == 2);
```

Comment:
Oops :) While debugging on IDE, I have to comment this out, otherwise the test fails during the debugging. Should have removed the comment before committing.